examples/patterns/navigation/dark.html

```diff
@@ -13,16 +13,16 @@
         </a>
       </div>
       <a href="#navigation" class="p-navigation__toggle--open" title="menu">Menu</a>
-      <a href="#navigation-closed" class="p-navigation__toggle--close" title="close menu">Menu</a>
+      <a href="#navigation-closed" class="p-navigation__toggle--close" title="close menu">Close Menu</a>
```

Comment:
I see, good point. Let's leave it as a link for now but I believe we would be better off using JavaScript to toggle the menu in the future.